core/tests/snapshot-recovery-test/tests/snapshot-recovery.test.ts

```diff
@@ -226,7 +227,7 @@ describe('snapshot recovery', () => {
         externalNodeLogs = await fs.open('snapshot-recovery.log', 'w');
 
         const enableConsensus = process.env.ENABLE_CONSENSUS === 'true';
-        let args = ['external-node', '--', '--enable-snapshots-recovery'];
```

Comment:
`--` shouldn't be removed because it separates args passed to the `zk` command and args passed to the EN binary. Because of this, [the test now fails](https://github.com/matter-labs/zksync-era/actions/runs/8500224833/job/23281855136?pr=1418#step:17:8) if `ENABLE_CONSENSUS` is specified.